.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -28,4 +28,5 @@ __Environment__
 
 - OS: [e.g. MacOS 10.2, Windows 10]
 - Camunda Modeler Version: [e.g. 2.0.0]
+- Execution Platform: [e.g. Camunda Platform, Camunda Cloud]
```

Comment:
Could also consider to add it to our feature request template.

Reply:
I added a note to the feature request template. We don't have a similar structure there, because OS and Version related stuff is probably not noteworthy.